kalite/coachreports/static/js/coachreports/coach_reports/views.js

```diff
@@ -30,6 +30,7 @@ var CoachSummaryView = BaseView.extend({
         _.bindAll(this, "set_data_model", "render");
         this.listenTo(this.model, "change:facility", this.set_data_model);
         this.listenTo(this.model, "change:group", this.set_data_model);
+        this.set_data_model();
```

Comment:
@benjaoming I got a merge conflict in here as well, perhaps @rtibbles can weigh in real quick?